content/docs/04-clusters/03-edge.md

```diff
@@ -51,6 +51,14 @@ Palette Edge Platform provides two solutions for edge, designed to support large
 

```

Comment:
@resmaa let's remove the beta tag

![CleanShot 2022-12-02 at 08 41 24](https://user-images.githubusercontent.com/29551334/205330556-cfe40f40-a626-4975-927f-aeff3c4df683.png)